apps/frontend/core-dapp/src/features/deposit/DepositSummary.tsx

```diff
@@ -24,6 +32,19 @@ const DepositSummary: React.FC = () => {
           }).format(estimatedReceivedAmount)}`
         )}
       </SummaryRecord>
+
+      {depositFees !== undefined && ppoReward !== undefined && (
+        <SummaryRecord label="PPO Reward" tooltip={<PPOReward />}>
+          <PPORewardAmount>
+            {Intl.NumberFormat(undefined, {
+              maximumFractionDigits: 2,
+              minimumFractionDigits: 1,
+              signDisplay: 'exceptZero',
+            }).format(+ppoReward)}
+          </PPORewardAmount>
+          &nbsp; ({toUsd(depositFees)})
+        </SummaryRecord>
+      )}
```

Comment:
And refactor this to:

```suggestion
      {(ppoReward === undefined || +ppoReward > 0) && (
        <SummaryRecord label="PPO Reward" loading={ppoReward === undefined || depositFees === undefined} tooltip={<PPOReward />}>
          <PPORewardAmount>
            {Intl.NumberFormat(undefined, {
              maximumFractionDigits: 2,
              minimumFractionDigits: 1,
              signDisplay: 'exceptZero',
            }).format(+(ppoReward ?? '0'))}
          </PPORewardAmount>
          &nbsp; ({toUsd(depositFees ?? 0)})
        </SummaryRecord>
      )}
```

Note that the `ppoReward ?? 0` and `depositFees ?? 0` don't matter because when we pass loading as true (when ppoReward or depositFees is undefined), the UI is overriden by a loading skeleton.